svc-bgs-api/src/config/setup.rb

```diff
@@ -0,0 +1,49 @@
+# https://www.cloudbees.com/blog/writing-microservice-in-ruby
+# This is the file that we’ll always load initially (even in the spec_helper.rb),
+# so it’s desirable that it doesn’t do too much (or your tests will be slow).
+
+# https://blog.eq8.eu/til/ruby-logs-and-puts-not-shown-in-docker-container-logs.html
+def redirect_output(prefix = nil)
+  $fdio_prefix = prefix
+  fd = IO.sysopen("/proc/1/fd/1", "w")
+  $fdio = IO.new(fd, "w")
+  $fdio.sync = true # send log message immediately
+  $fdio.puts "Docker should see this `puts` output"
+  def puts(args)
+    $fdio.puts("#{$fdio_prefix} #{args}")
+  end
+  $fdio
+end
+
+if(ENV['DOCKER_LOGS'] == '1' && File.exist?('/proc/1/fd/1'))
+  redirect_output("| ")
+end
+
+puts "Running setup.rb"
+
+
```

Comment:
I'd expect a linter to remove 2 consecutive blank lines. Can we add a Ruby linter, maybe as a follow-on ticket?